db/migrate/20220127141658_set_body_html_null_false.rb

```diff
@@ -0,0 +1,7 @@
+class SetBodyHtmlNullFalse < ActiveRecord::Migration[7.0]
+  def change
+    execute "UPDATE messages SET body_html = '' WHERE body_html IS NULL"
```

Comment:
I've added this here to make sure the database is migratable for local development environments, else it will complain about `body_html` having null values and will stop the migration. The downside of this is that people who only run this migration won't have any message bodies, because they're just set to an empty string here.

How do you feel about doing the data migration that's in the Rake task in the database migration? I'm not against it for simple stuff. The big upside is that it makes life a bit easier for other developers working on the app. The downside is that it can get stale and break eventually. WDYT?

Reply:
The alternative is that people just reload the seeds from this point on, could be viable as well. Not sure how to properly communicate that though :thinking:

Reply:
Let's remove it. If it is only for the development environment we can communicate that to folks.

Also, there's a single message in the seeds right now. So I don't think folks are relying to seeing a lot of them appear.